Timer/force-app/main/default/lwc/timerLWC/timerLWC.js

```diff
@@ -0,0 +1,52 @@
+import { LightningElement } from 'lwc';
+
+export default class TimerLWC extends LightningElement {
+
+    showStartBtn = true;
+    timeVal = 60;
+    mints = 5;
+    timeIntervalInstance;
+
+    start(event) {
+        this.showStartBtn = false;
+         var parentThis = this;
```

Comment:
We should never use var for declaring any variable.
Why do we need to assign this to a variable?